sktime/datasets/tests/test_uschange.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3 -u
+# coding: utf-8
+# copyright: sktime developers, BSD-3-Clause License (see LICENSE file)
+
+__author__ = ["Sebastiaan Koel"]
+__all__ = []
+
+import pandas as pd
+from sktime.datasets import load_uschange
+
+
+def check_dataset(checks, y=None, X=None):
+    """
+    asserts if datasets are loaded correctly
+    ----------
+    checks: dictionary with values to assert against should contain:
+        'columns' : list with column names in correct order,
+        'len_y'   : lenght of the y series (int),
+        'len_X'   : lenght of the X series/dataframe (int),
+        'data_types_X' : dictionary with column name keys and dtype as value,
+        'data_type_y'  : dtype if y column (string)
+    y:  y series of the dataset
+    X:  X dataframe or series from dataset
+    """
+    if y is not None:
+        assert(isinstance(y, pd.Series))
```

Comment:
I've only seen `assert` being used as a statement rather than function, please replace here and throughout, or is this a new thing? 
```suggestion
        assert isinstance(y, pd.Series)
```